packages/stream/create-stream/create-stream.js

```diff
@@ -0,0 +1,114 @@
+import { Internal } from '../../internal/index.js'
+import { InitializationInstance } from '../../app/init/init.js'
+
+/**
+ * Function to cek if the input text has a space or not. If yes, then will replace space with "-"
+ *
+ * @function
+ * @param {string} text -- stream's name
+ * @returns {string} -- stream's slug as per stream's name but without space
+ */
+function replaceWhiteSpace(text) {
+  return /\s/.test(text) ? text.split(' ').join('-') : text
+}
+
+/**
+ * @typedef Config
+ * @property {string} name -- name of stream
+ * @property {string} slug -- slug of stream
+ * @property {string} description -- description of stream
+ */
+
+/**
+ * @typedef FetchResponse
+ * @property {object} status -- A status response
+ * @property {object} data -- A return data as per endpoint
+ */
+
+/**
+ * A create stream module that should pass stream's name parameter, and user could pass the stream's slug & description parameter as well
+ *
+ * @function
+ * @param {InitializationInstance} initObject -- initialization object
+ * @param {Config} config - passing param from user including name, description, slug
+ * @returns {Promise<FetchResponse>} returns the restructured data which content status & created stream data
+ * @throws {Error}
+ */
+export const createStream = async (initObject, config) => {
+  if (!(initObject instanceof InitializationInstance) || !config) {
+    throw new TypeError(
+      'Failed to process because initialization is not valid and/or missing config. Please provide required initialization argument which is the initialization instance returned by the init() function and the configuration object'
```

Comment:
Please always be consistent when you describe something in multiple places. In the get list stream module, this part will throw an error with this description 

> Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function

This part has slightly different description, more like an improvement version. But still it's not consistent. 

Maybe we need to centralized the validations and errors in one place in the future, so our validation and error can be more consistent

Reply:
@faiq-naufal well noted mas, yeah we need to have same consistent error message regarding the initObject.
On commit [eb7cf9f](https://github.com/inlivedev/inlive-js-sdk/pull/11/commits/eb7cf9f994b1e43f9491d4583b2b3496c134ad6e), I've revised it same to be `Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function`. And after rethink about the config error message, also can be combine with config object format error message too (because it same checking if the config is there or not).

Kindly advice if it's okay for now if there's any other suggestion.